src/Nether.Analytics.DataLake/Scheduler/JobScheduleBase.cs

```diff
@@ -0,0 +1,72 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.WindowsAzure.Storage.Blob;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Nether.Analytics.DataLake
+{
+    public class EveryHourOnTheHourSchedule : JobScheduleBase
```

Comment:
Ahhh, just noticed these classes. This is probably where we should put the logic of calculating next time to execute the job. By doing so, we don't need to implement all schedules, because it can be implemented later by someone else with a specific need.